device-types/Cisco/FPR1K-DT-RACK-MNT.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Cisco
+model: FPR1K-DT-RACK-MNT
+slug: cisco-fpr1k-dt-rack-mnt
+part_number: FPR1K-DT-RACK-MNT
+u_height: 2.0
+is_full_depth: true
+subdevice_role: parent
+airflow: front-to-rear
+comments: '[Rack Mount Installation guide](https://www.cisco.com/c/en/us/td/docs/security/firepower/1010/hw/guide/hw-install-1010/mount.html)'
+weight: 900.0
+weight_unit: g
+is_powered: false
+device-bays:
+  - name: slot-1
+  - name: slot-2
+  - name: slot-3
+  - name: slot-4
```

Comment:
I believe there are only 2 bays for this, not 4.  You would not count the slots for mounting the power supply.